src/components/Common/ButtonAsIconWrapper.js

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+
+const ButtonAsIconWrapper = styled.button.attrs({
+  type: 'button'
+})`
+  min-width: ${props => remcalc(props.theme.elementSizes.tappableArea)};
```

Comment:
I'd move these two rules into our `UnstyledButton` and just add the centering directly  in the place where this component was being used